[Doc] Fix add Roboto font integration step to vite tutorial (#8419)
marmelab/react-admin: docs/Tutorial.md

}
```

> Lastly, add the `Roboto` font to the `index.html` file:

Reviewer: I'd put the custom css and the font after the initial screenshot, as it's strictly speaking not necessary to get the welcome screen (and it is more readable like that)

> In order to have the project using `Roboto` font, you have to add it to your index.html file

Reviewer 2: ```suggestion
Lastly, add the `Roboto` font to the `index.html` file:
```
